logic/src/androidMain/kotlin/com/wire/kalium/logic/util/DateTimeUtil.kt

```diff
@@ -2,6 +2,6 @@ package com.wire.kalium.logic.util
 
 import java.text.SimpleDateFormat
 
-private val serverDateTimeFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSXXX")
+private val serverDateTimeFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'")
```

Comment:
Probably not intended in this PR?

Whatever this is being used for, it would be nice to have tests making sure we can read the server time.

Reply:
you are right hehe! Not in the scope of the PR, just got a little ambitious having kalium compiling at last xd
I revert that commit =)